Makefile

```
@@ -114,7 +114,7 @@ re: clean all
 #---------------------------------------------------------------------------------
 
 test: all
-	@cp $(OUTPUT).3gx $(CITRA_DIR)/spicerack/bin/
+	@cp $(OUTPUT).3gx $(CITRA_DIR)/spicerack/bin/Saltwater.3gx
```

Continuing from #4:
> > replace with `$(OUTPUT).3gx`
> this was intended, barista is hard coded to look for `/spicerack/bin/Saltwater.3gx`, not `$(OUTPUT).3gx`.
> 
> I have my fork under `SaltwaterMyFork` which means it will generate a `SaltwaterMyFork.g3x`
That is completely my bad, however, I have a better idea on how to do this in that case
```suggestion
	@cp $(OUTPUT).3gx $(CITRA_DIR)/spicerack/bin/
```

---

continuing that suggestion since i can't comment on unchanged lines

```diff
[line 12]
- TARGET := $(notdir $(CURDIR))
+ TARGET := Saltwater
```